### PR TITLE
adds test for payload client/daemon version validation

### PR DESCRIPTION
**Pull request by zacharym**
@jsingle cc @thieman

### CODE REVIEW

File: dusty/payload.py [hunk 1]
```diff
@@ -13,6 +15,8 @@ def __eq__(self, other):
         return False
 
     def run(self):
+        if self.client_version != VERSION:
+            raise RuntimeError("Dusty daemon is running version: {}, and client is running version: {}".format(VERSION, self.client_version))
```
Comment (jsingle) [inline dusty/payload.py]: this is not the method the daemon uses to actually run the Payload.  This is just a helper function, which is actually just used by the client ATM.  Check the daemon code to see how the daemon should check the client_version

File: dusty/payload.py [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import cPickle
+from constants import VERSION
```
Comment (thieman) [inline dusty/payload.py]: Relative imports always

**Comment by zacharym** [comment]
@jsingle @thieman this is updated

**Comment by jsingle** [comment]
:rocket: assuming you've tested it

**Pull request merged**